source/agora/test/Base.d

```diff
@@ -40,6 +40,7 @@ import agora.consensus.data.Transaction;
 import agora.consensus.data.UTXOSet;
 import agora.consensus.EnrollmentManager;
 import agora.consensus.Genesis;
+import VGb = agora.consensus.validation.GenesisBlock;
```

Comment:
Why? Just import `agora.consensus.validation` and call `isValid` on the block.

Reply:
I remove it.